lib/ship_quiet_logistics.rb

```diff
@@ -13,6 +18,63 @@
 require 'ship_quiet_logistics/uploader'
 require 'ship_quiet_logistics/version'
 
+AWS.config(access_key_id: ENV['QUIET_AWS_ACCESS_KEY'],
```

Comment:
It would also certainly, what's a way to avoid this? I'm thinking I should be called `AWS::SQS.new` with the creds same with S3... Also would remove the awkwardness of having this be global.